autogpt_platform/backend/backend/util/retry.py

```diff
@@ -21,28 +22,33 @@ def _log_prefix(resource_name: str, conn_id: str):
 def conn_retry(resource_name: str, action_name: str, max_retry: int = 5):
     conn_id = str(uuid4())
 
-    def before_call(retry_state):
-        prefix = _log_prefix(resource_name, conn_id)
-        logger.info(f"{prefix} {action_name} started...")
-
-    def after_call(retry_state):
-        prefix = _log_prefix(resource_name, conn_id)
-        if retry_state.outcome.failed:
-            # Optionally, you can log something here if needed
-            pass
-        else:
-            logger.info(f"{prefix} {action_name} completed!")
-
     def on_retry(retry_state):
         prefix = _log_prefix(resource_name, conn_id)
         exception = retry_state.outcome.exception()
         logger.info(f"{prefix} {action_name} failed: {exception}. Retrying now...")
 
-    return retry(
-        stop=stop_after_attempt(max_retry + 1),
-        wait=wait_exponential(multiplier=1, min=1, max=30),
-        before=before_call,
-        after=after_call,
-        before_sleep=on_retry,
-        reraise=True,
-    )
+    def decorator(func):
```

Comment:
There is a bug on our retry decorator that we don't print `completed successfully`.
This decorator is used for acquiring db client.